### PR TITLE
Added a workaround for an old analyzer error. It goes away with ddk (new

### PR DESCRIPTION
**Pull request by efortuna**
Added a workaround for an old analyzer error. It goes away with ddk (new analyzer versions)

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: crypto
-version: 2.1.0
+version: 2.2.0
```
Comment (natebosch) [inline pubspec.yaml]: How about `2.1.1` for a bug fix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.2.0
+  * Added a workaround for an old analyzer error. It goes away with ddk (new
+    analyzer versions).
```
Comment (natebosch) [inline CHANGELOG.md]: [nit] It isn't a new analyzer version - it's switching to CFE away from analyzer. I'd phrase this as.

"Added a workaround for a bug in DDC (used in build_web_compilers 1.x). This bug is not present in DDK (used in build_web_compilers 2.x)"

File: lib/src/sha512.dart [hunk 3]
```diff
@@ -6,6 +6,7 @@ import 'dart:convert';
 
 import 'digest.dart';
 import 'hash.dart';
+// ignore: uri_does_not_exist
 import 'sha512_fastsinks.dart' if (dart.library.js) 'sha512_slowsinks.dart';
 //import 'sha512_slowsinks.dart';
```
Comment (natebosch) [inline lib/src/sha512.dart]: [optional] If you'd like to clean this up while in this file anyway - a commented out import should have not been submitted with the last PR.

Comment (efortuna) [reply lib/src/sha512.dart]: done

File: CHANGELOG.md (hunk 2 above)
Comment (efortuna) [reply CHANGELOG.md]: done

File: pubspec.yaml (hunk 1 above)
Comment (efortuna) [reply pubspec.yaml]: done

**Pull request merged**